subsys/ipc/ipc_service/backends/Kconfig.icmsg

```diff
@@ -1,10 +1,17 @@
 # Copyright (c) 2022 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-if IPC_SERVICE_BACKEND_ICMSG
+config ICMSG
+	bool
+	select SPSC_PBUF
+	select SPSC_PBUF_USE_CACHE
+	help
+	  If icmsg library is to be included in the build
+
+if ICMSG
```

Comment:
Uhm, the nomenclature was consistently `IPC_SERVICE_*` for backends and libraries. If we now decide to move away from that for libraries we should also fix the other libraries we have in the tree, like the RPMsg support library.

Reply:
Good point. I'll add `IPC_SERVICE_` prefix to the labels.